src/sentry/message_filters.py

```diff
@@ -1,38 +1,11 @@
-# TODO RaduW 8.06.2019 remove the sentry.filters package and rename this module to filters
 from __future__ import absolute_import
 
-import collections
-from collections import namedtuple
-import re
-
 from rest_framework import serializers
-from six.moves.urllib.parse import urlparse
-from ua_parser.user_agent_parser import Parse
 
 from sentry.api.fields.multiplechoice import MultipleChoiceField
 from sentry.models.projectoption import ProjectOption
 from sentry.signals import inbound_filter_toggled
 from sentry.utils.data_filters import FilterStatKeys, get_filter_key
-from sentry.utils.safe import get_path
-
-
-EventFilteredRet = namedtuple("EventFilteredRet", "should_filter reason")
-
-
-def should_filter_event(project_config, data):
-    """
-    Checks if an event should be filtered
-
-    :param project_config: relay config for the request (for the project really)
-    :param data: the event data
-    :return: an EventFilteredRet explaining if the event should be filtered and, if it should the reason
-        for filtering
-    """
-    for event_filter in get_all_filters():
-        if _is_filter_enabled(project_config, event_filter) and event_filter(project_config, data):
-            return EventFilteredRet(should_filter=True, reason=event_filter.spec.id)
-
-    return EventFilteredRet(should_filter=False, reason=None)
 
 
 def get_all_filters():
```

Comment:
Maybe we should change the name to `get_all_filter_specs` or something like that.
At any rate we should change the doc string (since we now return FilterSpecs not predicates)

Reply:
Does the pushed change work for you? I feel like we need to clean this up a bit sometime, given that the business logic disappeared I feel like some of the abstraction in here serves no purpose.